packages/ui-extensions-server-kit/.gitignore

```diff
@@ -1,3 +1,5 @@
 *.d.ts
 !typings.d.ts
+!index.d.ts
+!testing.d.ts
```

Comment:
What is this for?

Reply:
These are just entry points pointing to the build.